### DISCUSSION TITLE
What to put for `OPENTOGETHERTUBE_API_KEY` and `SESSION_SECRET`?

### DISCUSSION THREAD

**Discussion by shamith-ventura** [dyc3/opentogethertube#850] [Q&A] [answered]
`OTT_HOSTNAME=localhost:8080
LOG_FILE=./logs/ott.log
LOG_LEVEL=info
OPENTOGETHERTUBE_API_KEY=GENERATE_YOUR_OWN_API_KEY
SESSION_SECRET=GENERATE_YOUR_OWN_SECRET
YOUTUBE_API_KEY=
GOOGLE_DRIVE_API_KEY=API_KEY_GOES_HERE
DISCORD_CLIENT_ID=NONE
DISCORD_CLIENT_SECRET=NONE
GOOGLE_CLIENT_ID=NONE
GOOGLE_CLIENT_SECRET=NONE
DB_MODE=sqlite
ADD_PREVIEW_SEARCH_MIN_LENGTH=3`

Above is the current env and I'm unable to figure out what to put for the following key's meaning how do I acquire it.

OPENTOGETHERTUBE_API_KEY
SESSION_SECRET

**Comment by dyc3** [comment, marked as answer]
As the string `GENERATE_YOUR_OWN_SECRET` would imply, generate them yourself. Go grab a password generator.

Note that `OPENTOGETHERTUBE_API_KEY` requires the value to be >= 40 chars, and `SESSION_SECRET` requires >= 80 chars.
